bindings/java/src/org/sleuthkit/datamodel/Report.java

```diff
@@ -236,6 +237,11 @@ public BlackboardArtifact newArtifact(int artifactTypeID) throws TskCoreExceptio
 		return db.newBlackboardArtifact(artifactTypeID, objectId);
 	}
 
+	@Override
+	public AnalysisResult newAnalysisResult(BlackboardArtifact.Type artifactType, Score score, String conclusion, String configuration, String justification, Collection<BlackboardAttribute> attributesList) throws TskCoreException {
```

Comment:
If we allow them to make artifacts on reports, we can let them make analysis results I think.